src/composables/routes.js

```diff
@@ -94,6 +94,9 @@ export default memoizeForContainer(({ router, requestData }) => {
     return `/projects/${projectId}/forms/${encodedFormId}/submissions/${encodedInstanceId}`;
   };
 
+  const editSubmissionPath = (projectId, xmlFormId, instanceId) =>
```

Comment:
Instead of adding a new function `editSubmissionPath()`, how about adding an optional `suffix` parameter to `submissionPath()`? We do something like that for `projectPath()` and `formPath()`.